server/src/modules/user/user.controller.ts

```diff
@@ -1,11 +1,24 @@
-import { Body, Controller, Get, Param, Put } from '@nestjs/common';
+import { Body, Controller, Get, Param, Post, Put } from '@nestjs/common';
 import { UserService } from './user.service';
 import { User } from './user.model';
 
+export interface CreateUserDTO {
+  firstName: string;
+  lastName: string;
+  email: string;
+  password: string;
+  bio: string;
+  userType: string;
+}
 @Controller('mentorship')
 export class UserController {
   constructor(private userService: UserService) {}
 
+  @Post('user-create')
+  async createUser(@Body() user: User) {
+    return this.userService.createUser(user);
```

Comment:
nit: this isn't especially secure: we're taking a @Body object without validating it (it could contain any keys/values from the user), and then the `createUser` method takes `user: any` and calls `this.userMode.create` given that input: this allows the user to set ANY fields they want on their model (maybe they could even make a fake created-at or modified-at date, or do other thing they shouldn't).

Not a huge deal for now! But in a real-world app it might be a security concern. If you have extra time and want to practice making it secure, something like ValidationPipe in NestJS can help (https://docs.nestjs.com/techniques/validation#transform-payload-objects)